testutils/collection/data.go

```diff
@@ -737,7 +737,7 @@ var (
 
 	ByteFileShareSnapshot = `{
 		"id": "3769855c-a102-11e7-b772-17b880d2f537",
-		"name": "sample-snapshot-01",
```

Comment:
any reason for removing "-" in between?

Reply:
same reason as above that no special characters are not allowed